main/inc/lib/chat.lib.php

```diff
@@ -244,6 +244,26 @@ public function sanitize($text)
         $text = str_replace("\n", "<br>", $text);
         return $text;
     }
+    
+    public static function  set_disable_chat($status)
```

Comment:
If this one is static, disable_chat() should be as well, no?
$status could have a default value of "true" considering the function is called "set_disable_chat".
New functions created should be inUpperCaseLetters().
There are *two* spaces between "function" and "set_disable_chat".
There is no documentation block:
/**
 * Disables or enables the chat
 * @param bool $status True to disable, false to enable the chat
 */